cli.js

```diff
@@ -83,6 +83,7 @@ if ("--help" in cliFlags) {
     console.log("   --iteration-count:   Set the default iteration count.");
     console.log("   --worst-case-count:  Set the default worst-case count");
     console.log("   --dump-json-results: Print summary json to the console.");
+    console.log("   --dump-test-list:    Print test list.");
```

Comment:
Nit: might be worth adding "instead of running"

Reply:
Sounds good! Changed.